TeamCode/src/main/java/org/firstinspires/ftc/teamcode/freightfrenzy2021/opmodes/AutonOpModeV2.java

```diff
@@ -140,4 +152,39 @@ private void updateTelemetry(){
         telemetry.update();
     }
 
+    //Open up the camera for Freight Detection
+    private void startCamera(){
+        freightDetector = new FreightDetector();
+
+        //No Longer needed when using overloaded function createWebcam(webCabName)
+        //int cameraMonitorViewId = hardwareMap.appContext.getResources().getIdentifier("cameraMonitorViewId", "id", hardwareMap.appContext.getPackageName());
+        //Log.d(logTag, "cameraMonitorViewId set");
+        EbotsWebcam bucketWebCam = new EbotsWebcam(hardwareMap, "bucketCam", RobotSide.FRONT, 0,-3.25f, 9.0f);
+        WebcamName webcamName = bucketWebCam.getWebcamName();
+        // With live preview
+        //why did we change from createWebcam(webCabName, viewportContainerId) as to createWebcam(webCabName)
```

Comment:
The single argument constructor doesn't pass the video feed to the Driver Station, which served as the viewportContainer previously.